build.gradle

```diff
@@ -1351,6 +1351,50 @@ project(':tools') {
   }
 }
 
+project(':shell') {
+  archivesBaseName = "kafka-shell"
+
+  dependencies {
+    compile libs.argparse4j
+    compile libs.jacksonDatabind
+    compile libs.jacksonJDK8Datatypes
+    compile libs.jline
+    compile libs.slf4jApi
+    compile project(':clients')
+    compile project(':core')
+    compile project(':log4j-appender')
+    compile project(':metadata')
+    compile project(':raft')
+
+    compile libs.jacksonJaxrsJsonProvider
+
+    testCompile project(':clients')
+    testCompile libs.junitJupiter
+    testCompile project(':clients').sourceSets.test.output
```

Comment:
Do we actually need this here? This would only be needed if there was a class in clients/src/test that we depend on. Doing this "output" kind of dependency in gradle can slow down the build, so we should avoid if possible

Reply:
hmm, seems it's not needed.  I forget why I thought I needed it.  I'll remove it.